bentoml/_internal/server/grpc_app.py

```diff
@@ -128,21 +132,32 @@ def interceptors(self) -> list[aio.ServerInterceptor]:
             AsyncOpenTelemetryServerInterceptor as AsyncOtelInterceptor,
         )
 
-        # TODO: prometheus interceptors.
-        interceptors: list[t.Type[aio.ServerInterceptor]] = [
-            GenericHeadersServerInterceptor,
-            AsyncOtelInterceptor,
-        ]
+        interceptors: list[aio.ServerInterceptor] = []
+
+        if self.enable_metrics:
+            from .grpc.interceptors.prometheus import PrometheusServerInterceptor
+
+            prometheus_interceptor = PrometheusServerInterceptor(
+                bento_service=self.bento_service
+            )
```

Comment:
You can make a partial function for this.
```suggestion
            prometheus_interceptor = partial(PrometheusServerInterceptor, bento_service=self.bento_service)
```
import partial from functools.